packages/server/graphql/intranetSchema/mutations/updateEmail.ts

```diff
@@ -60,6 +64,7 @@ const updateEmail = {
         .run(),
       updateUser(updates, userId)
     ])
+    await dataLoader.get('users').clear(userId)
```

Comment:
What's the reason for that change? 🤔

Reply:
Mutations updateWatchlist and deleteUser now clear the user from dataloader to return correct result on next query.

Reply:
I checked again. Actually this just worked around the problem. `User` is stored in `LocalCache` and this call would clear it from that. So the real issue was a missing `db.write('User', userId, updates)` call. All calls to `updateUser` need to be paired with that and it was missing in `updateEmail` and `updateWatchlist`.
It would be best to combine those two calls into one, but putting it into `updateUser` seems like not the right place since it would mix db writes with the cache. I think long term it would be better to do reads and writes through `db` and call PG from there just like we do with the reads.

Reply:
I think for transitioning `User` away from RethinkDB it's best to extend `customRedisQueries` to writes as this can then be used to write to PG instead of RethinkDB, just like I plan to do with reads: https://github.com/ParabolInc/parabol/blob/aac85bf13878746968236ee32c134fb9a4d6f0be/packages/server/dataloader/customRedisQueries.ts#L61
This would then make the direct `updateUser` call superfluous and avoid this type of error.